### PR TITLE
[opensuse] KMP check

### PR DESCRIPTION
**Pull request by scarabeusiv**
So far just quickly ported, I need to review what it does and add testcases.

### CODE REVIEW

File: rpmlint/checks/KMPPolicyCheck.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+
+class KMPPolicyCheck(AbstractCheck):
+    def check(self, pkg):
+        if pkg.isSource() or pkg.name.find('-kmp-') < 0:
+            return
+
+        pkg_requires = set(map(lambda x: x[0].split('(')[0], pkg.requires()))
+
+        kernel_flavour = "kernel-" + pkg.name.partition('-kmp-')[2]
```
Comment (Conan-Kudo) [inline rpmlint/checks/KMPPolicyCheck.py]: Please consistently use `flavor` here (American English is used throughout rpmlint)

Comment (scarabeusiv) [reply rpmlint/checks/KMPPolicyCheck.py]: sure.

**Comment by Conan-Kudo** [comment]
@scarabeusiv Why drop the `suse-` prefix? This is specific to SUSE kernel module packaging, since SUSE doesn't follow the standard kmod packaging convention.

**Comment by scarabeusiv** [comment]
Because it is in our branch anyway and we don't prefix the others either as it eats quite bit space on the rpmlint output line.

**Pull request merged**